### DISCUSSION TITLE
Need help with Spacing attributes

### DISCUSSION THREAD

**Discussion by rahulpd101** [ng-matero/ng-matero#607] [Q&A] [answered]
Hi,
I am using a padding class attributes to style my div.
I was wondering if there is any better way to use these styles together. This is what I am doing:
`<div class="col-md-12 p-r-16 p-l-16 p-t-16">`
What if I could it like this?
`<div class="col-md-12 p-r-l-t-16">`

**Comment by nzbin** [comment, marked as answer]
`p-r-16` + `p-l-16` = `p-x-16`
`p-t-16` + `p-b-16` = `p-y-16`
`p-t-16` + `p-b-16` + `p-r-16` + `p-l-16` = `p-16`
